src/kirby/components/list/list.component.ts

```diff
@@ -20,14 +21,15 @@ import {
 } from './list.directive';
 import { LoadOnDemandEvent, LoadOnDemandEventData } from './list.event';
 import { ListHelper } from './helpers/list-helper';
+import { ListRowClassService } from '~/kirby/components/list/list-row-class.service';
 
 export type ListShape = 'square' | 'rounded';
 
 @Component({
   selector: 'kirby-list',
   templateUrl: './list.component.html',
   styleUrls: ['./list.component.scss'],
-  providers: [ListHelper],
+  providers: [ListHelper, ListRowClassService],
```

Comment:
Maybe `ListRowClassHelper`? Or merged into `ListHelper`?

Reply:
I think the words "Helper" and "Utils" are the most abused (and horrific) names for constructs in software.

`ListHelper` says that it "helps the list"... but, to do what? I've (previously) experienced that a "Helper"-class becomes a box of arbitrary tools with no to little "low coupling, high cohesion" 

I'm not opposed to renaming `ListRowClassService` to something else, just not sure of a better alternative... any suggestions?

Reply:
Lets keep it as is. I cannot come up with anything better and it is a service for getting css classes for rows. The best reason for renaming would be to align naming the naming to match the ListHelper.